model/dialogue/hmnet/config/dialogue_conf

```diff
@@ -0,0 +1,95 @@
+##################
```

Comment:
Maybe change the name of this file to `hmnet.conf`? It's generally not a good idea to have a file without extension name